services/user.py

```diff
@@ -0,0 +1,40 @@
+from db.models import User
+
+
+def create_user(
+        username: str, password: str, email: str = None,
+        first_name: str = None, last_name: str = None
+) -> None:
+
+    user = User.objects.create_user(username=username)
+    user.set_password(raw_password=password)

```

Comment:
It's possible to set password directly in `.create_user()`.

Reply:
Thanks, changed.